gdal/frmts/grok/eoptemplate_pleiades.xml

```diff
@@ -0,0 +1,46 @@
+<eop:EarthObservation
```

Comment:
It is unclear to me that this needs to be duplicated since it is already available in gdal/frmts/openjpeg.  Also, it's use is not mentioned in the rst file.

Reply:
Thanks, removed.